### PR TITLE
Tiny clean up model and column package.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: util/types/convert.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func convertIntToInt(val, lowerBound, upperBound int64, tp byte) (converted int6
 	return
 }
 
-func convertToInt(val interface{}, target *FieldType) (converted int64, err error) {
+func convertToInt(val interface{}, target FieldType) (converted int64, err error) {
```
Comment (shenli) [inline util/types/convert.go]: Why not use pointor?

Comment (qiuyesuifeng) [inline util/types/convert.go]: I see FieldType variable is only used for read variable, also in ColumnInfo struct with no pointer FieldType,  so i think it is better to use a none-pointer variable here.

Comment (shenli) [inline util/types/convert.go]: I am concerned about performence issue.

Comment (qiuyesuifeng) [inline util/types/convert.go]: Gotcha, u are right.

**Pull request closed without merging**